pkg/backend/openshiftcluster/create.go

```diff
@@ -124,6 +125,16 @@ func (m *Manager) Create(ctx context.Context) error {
 		},
 	}
 
+	pullSecret, err := pullsecret.Merge(os.Getenv("PULL_SECRET"), string(m.doc.OpenShiftCluster.Properties.ClusterProfile.PullSecret))
```

Comment:
So this env pull secret will be there for dev getting the secrets for pulling in for repositories we need to setup the cluster.  What is the story in Prod/INT will it be on the customer to have a pull secret for every repo we use or will we have a core set we add onto their pull secret?